asctb-api/src/functions/graph.functions.ts

```diff
@@ -24,7 +25,7 @@ export function buildgraphAS(data: Row[], graphData: GraphData) {
 
     row.anatomical_structures.forEach((structure) => {
       let s: number;
-      if (structure.id) {
+      if (structure.id && structure.id.toLowerCase() !== 'not found') {
```

Comment:
We should probably clear out 'not found' ids using the fixOntologyId function https://github.com/hubmapconsortium/ccf-asct-reporter/blob/addingBMTypeToGraphOut/asctb-api/src/functions/lookup.functions.ts#L15